src/routes.ts

```diff
@@ -0,0 +1,8 @@
+export const ROUTES = {
+  ROOMS_OVERVIEW: "/rooms",
+  HOME: "/",
```

Comment:
```suggestion
  HOME: "/",
  ROOMS_OVERVIEW: "/rooms",
```

triggering my inner OCD

Reply:
✅